plugins/shortcuts/src/api/ShortcutApi.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Observable from 'zen-observable';
+import { createApiRef } from '@backstage/core';
+import { Shortcut } from '../types';
+
+export const shortcutsApiRef = createApiRef<ShortcutApi>({
+  id: 'plugin.shortcuts.api',
+  description: 'API to handle shortcuts in a Backstage Sidebar',
+});
+
+export interface ShortcutApi {
+  /**
+   * Returns an Observable that will subscribe to changes.
+   */
+  observe(): Observable<Shortcut[]>;
```

Comment:
In the [catalog refactor](https://github.com/backstage/backstage/pull/5372) I saw a pattern for observables of `<thing>$()` so maybe `shortcuts$(): Observable<Shortcut[]>` here?

Reply:
👍